### PR TITLE
LFR-3192: Add Chip component for tags

### PR DESCRIPTION
**Pull request by zlahham**
Co-authored-by: Janie Amero <43782388+jamero102@users.noreply.github.com>

### CODE REVIEW

File: packages/core/src/globals/typography.css [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * ```
  */
 :root {
-  --font-avenir: 'Avenir Next W01', 'Helvetica Neue', 'Helvetica', 'sans-serif';
+  --font-avenir: 'Avenir Next', 'Helvetica Neue', 'Helvetica', 'sans-serif';
```
Comment (Denyer) [inline packages/core/src/globals/typography.css:16]: Where is this font coming from? the font on the site is 'Avenir Next W01', unless I'm missing something?

File: packages/core/src/components/Chip/Chip.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  background: var(--color-greyDarkest);
+  font-family: var(--font-avenir);
+  padding: 0.375rem 0.625rem;
+  text-decoration: none;
+  display: inline-block;
+  height: auto;
+  width: auto;
+  border: 1px solid var(--color-greyDarkest);
+  box-sizing: border-box;
+  font-weight: 600;
```
Comment (Denyer) [inline packages/core/src/components/Chip/Chip.css:12]: We don't support 600 font weight in Avenir, it's 400, 700 and 800, should this be:
```suggestion
  font-weight: var(--fontweight-demi);
```

File: packages/core/src/components/Chip/Chip.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  className?: string,
+  href: string,
+  text: string,
+};
+
+const Chip = ({className, href, text}: Props) => (
+  <a href={href} className={cx(css.root, className)}>
+    <div>
+      {text}
+    </div>
```
Comment (Denyer) [inline packages/core/src/components/Chip/Chip.js:18]: this div seems unnecessary:
```suggestion
    {text}
```

File: packages/core/src/components/Chip/Chip.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import * as React from 'react';
+import cx from "classnames";
+import css from "./Chip.css";
+
+type
+Props = {
+  className?: string,
+  href: string,
+  text: string,
+};
```
Comment (Denyer) [inline packages/core/src/components/Chip/Chip.js:12]:
```suggestion
type Props = {
  className?: string,
  href: string,
  text: string,
};
```

**Review by Denyer — CHANGES_REQUESTED** [review]
A couple of questions

File: packages/core/src/globals/typography.css (hunk 1 above)
Comment (zlahham) [reply packages/core/src/globals/typography.css:16]: oops, this was a mistake. Will amend now

File: packages/core/src/components/Chip/Chip.css (hunk 2 above)
Comment (zlahham) [reply packages/core/src/components/Chip/Chip.css:12]: Yeah true, will double check with Oli about this.

File: packages/core/src/components/Chip/Chip.js (hunk 4 above)
Comment (zlahham) [reply packages/core/src/components/Chip/Chip.js:12]: editor linting 🤦

File: packages/core/src/components/Chip/Chip.test.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import Chip from './Chip';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  render(
```
Comment (tblakey89) [inline packages/core/src/components/Chip/Chip.test.js:8]: Maybe worth testing the text is displayed?

**Review by Denyer — APPROVED** [review]

**Review by tblakey89 — APPROVED** [review]

**Pull request merged**